src/pages/TeamPage/TeamPage.tsx

```diff
@@ -198,7 +201,43 @@ export const TeamPage: React.FC = () => {
 
   // Option 1 - Remove user from Team
 
+  const canRemoveFromTeam = (member: FetchTeamMembersResponse) => {
+    const currentUser = user;
+
+    // If we do not have our current user's data, we do not allow for any sort of team member removal.
+    if (!currentUser) return false;
+
+    /* If our current user is either an admin or an org owner, and is NOT the only team member,
+    we allow for team member removal. */
+    if (AUTHORIZED_ROLES.includes(getUserMemberRole(currentUser).name) && members.length > 1) return true;
```

Comment:
This logic is not correct it needs to validate if the user is the last admin or org owner and not if he's the last member of the team.